src/calculateRentalCost.js

```diff
@@ -28,7 +28,23 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let sumRentals = days * 40;
+  const shortTerm = 3;
+  const longTerm = 7;
+  const rentalplusseven = 50;
```

Comment:
```suggestion
  const rentalPlusSeven = 50;
```
use camelCase in variables name

Reply:
corrected